hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/local/LocalRegionCacheTest.java

```diff
@@ -94,6 +95,16 @@ public void testThreeArgConstructorRegistersTopicListener() {
         verify(topic).addMessageListener(isNotNull(MessageListener.class));
     }
 
+    @Test
+    public void testTimestampToString() {
+        assertNotNull(new Timestamp().toString());
+    }
+
+    @Test
+    public void testInvalidationToString() {
+        assertNotNull(new Invalidation().toString());
+    }
+
     public static void runCleanup(LocalRegionCache cache) {
```

Comment:
I was refering to the other toString test methods. They don't make much sense since they just check for not null.